pyinjective/proto/exchange/injective_auction_rpc_pb2_grpc.py

```
@@ -96,6 +101,13 @@ def AccountAuctionsV2(self, request, context):
         context.set_details('Method not implemented!')
         raise NotImplementedError('Method not implemented!')
 
+    def AuctionAccountStatus(self, request, context):
```

Ruff (0.15.0)

[warning] Unused method argument: `request`

(ARG002)